samples/cellular/nrf_cloud_multi_service/prj.conf

```diff
@@ -106,6 +106,9 @@ CONFIG_DATE_TIME=y
 
 # LTE link control - used by PGPS and main application
 CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_LC_NCELLMEAS=y
+CONFIG_LTE_LC_PSM=y
+CONFIG_LTE_LC_XMODEMSLEEP=y
```

Comment:
Are these still necessary or does Location library handle this now with `imply`? Same for location sample.